syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
Fair point. @PProvost, do you have any thoughts? It's a regression in visuals but is more correct, which means it maps any themes closer to other, more common language syntaxes.